src/Security/OpenIdConfigurationProvider.php

```diff
@@ -404,6 +404,7 @@ private static function base64urlDecode(string $input): string
     {
         $decoded = base64_decode(strtr($input, '-_', '+/'));
 
+        /* @phpstan-ignore identical.alwaysFalse */
```

Comment:
If i understand https://www.php.net/manual/en/function.base64-decode.php correctly, `base64_decode` will never return `false` unless it is set to be strict. 

```suggestion
        $decoded = base64_decode(strtr($input, '-_', '+/'), true);
```